src/components/Layout/index.tsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
```

Comment:
I'm not sure if you're still working on this file. The children don't need to be typed, just use the FC type of React.
```
import React, { FC } from "react";
import { Header } from "../Header/Header";
import { Footer } from "../Footer";

export const Layout: FC = ({ children }) => (
  <>
    <Header />
    {children}
    <Footer />
  </>
);
```